slice.go

```diff
@@ -693,3 +693,93 @@ func Splice[T any, Slice ~[]T](collection Slice, i int, elements ...T) Slice {
 
 	return append(append(append(output, collection[:i]...), elements...), collection[i:]...)
 }
+
+// Pull removes all given values from splice using SameValueZero for equality comparisons.
+func Pull[T comparable, Slice ~[]T](collection Slice, elements ...T) Slice {
+	output := make(Slice, 0, len(collection))
+
+	for _, item := range collection {
+		shouldRemove := false
+		for _, element := range elements {
+			if item == element {
+				shouldRemove = true
+				break
+			}
+		}
+
+		if !shouldRemove {
+			output = append(output, item)
+		}
+	}
+
+	return output
+}
+
+// This method is like Pull except that it accepts an array of values to remove
```

Comment:
```suggestion
// PullAll is like [Pull] except that it accepts an array of values to remove
```

Here you would respect the convention about comment that must start with the method/function name. Also, you would link Pull with a godoc link

Reply:
Appreciate for your advice!